src/colvarbias.cpp

```diff
@@ -84,6 +84,9 @@ int colvarbias::init(std::string const &conf)
 
   get_keyval(conf, "outputEnergy", b_output_energy, b_output_energy);
 
+  // Disabled by default in base class; default value can be overridden by derived class constructor
```

Comment:
I would wrap this into `if (is_available(f_cvb_bias_actual_colvar) { ... }` to not confuse users of biases that don't implement it.

Reply:
Since this is `parse_silent`, it would not change the output if the option is not specified in the config. If the option is specified but not available, I think a "failed dependency" message is more informative than "unknown keyword". More generally, I think it's a limitation of parsing keywords conditionally to the context, that when not available they become completely unknown to the parser, and then we're unable to give precise feedback.

Reply:
On that I strongly agree.  I considered passing a bitwise flag through the `parse_mode` argument to `get_keyval()`, which is easy to implement but it will take time to adapt all the calls for sure.